prov/shm/src/smr_init.c

```diff
@@ -167,7 +167,6 @@ static int smr_getinfo(uint32_t version, const char *node, const char *service,
 				return -FI_ENODATA;
 			}
 			cur->domain_attr->mr_mode |= FI_MR_HMEM;
-			ofi_hmem_init();
```

Comment:
Would you need this for a DSO build?

Reply:
Yep - it will.  @aingerson - please see fi_verbs.c as an example.